### PR TITLE
Adapt top-level README to describe local environment setup

### PR DESCRIPTION
**Pull request by MichaelSchubertTNG**

### CODE REVIEW

**Review by jendrikjoe — COMMENTED** [review]
Why did you remove the `docker-compose` all together? 🙂

File: backend/flyway/build_flyway_image.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
-mkdir -p migration
-cp ../src/main/resources/db/migration/*.sql migration/
+mkdir -p migration_tmp
+cp ../src/main/resources/db/migration/*.sql migration_tmp/
 docker build . -t trackyourbed-flyway
```
Comment (jendrikjoe) [inline backend/flyway/build_flyway_image.sh]: Can we actually make the tag an argument.
If you don't wanna do that, I can take care of this in a separate MR 😉

**Review by jendrikjoe — CHANGES_REQUESTED** [review]

File: backend/flyway/build_flyway_image.sh (hunk 1 above)
Comment (MichaelSchubertTNG) [reply backend/flyway/build_flyway_image.sh]: That's a good suggestion, then the parameters the local setup relies on to be the same are closer to each other, I will add it here

Comment (MichaelSchubertTNG) [reply backend/flyway/build_flyway_image.sh]: pushed

Comment (jendrikjoe) [reply backend/flyway/build_flyway_image.sh]: Awesome 👍

**Review by jendrikjoe — APPROVED** [review]

**Pull request merged**